README.md

```diff
@@ -7,3 +7,34 @@ This repository holds the ReStructuredText (RST) source, and other files, for us
 ## Building Ansible community documentation
 
 Follow the documentation to [set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally) and then [build Ansible community documentation locally](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#building-the-documentation-locally)
+
+## Using nox
+
+This project includes a ``nox`` configuration to automate checks and other functions.
+We recommend that you use ``nox`` virtual sessions to run checks locally before you submit a pull request.
+
+Install ``nox``.
+
+    # Install nox with pip
+    python3 -m pip install nox
+
+    # Install nox on MacOS
+    brew install nox
+
+Run ``nox --list`` from the repository root to view available sessions.
+
+## Running the spelling check
+
+This repository uses ``codespell`` to check for common spelling mistakes in the documentation source.
+
+Run ``nox -s spelling`` to check spelling.
+
+Run ``nox -s spelling -- -w`` to correct spelling errors.
+
+If you want to run spelling checks without ``nox`` do the following:
+
+    # Install codespell
+    python3 -m pip install -r tests/spelling.in -c tests/spelling.txt
+
+    # Check spelling
+    codespell docs/docsite
```

Comment:
I would remove this part. I don't think we need to document two ways to do the same thing.

Reply:
Sounds good. I'll add a link to the codespell project page so users can figure out how to do their own thing if they wish.